src/index.html

```diff
@@ -1,13 +1,280 @@
+                    <img
+                      src="./images/categories-image3.png"
+                      alt="Audio"
+                      class="category__photo">
+                  </a>
+                  <a href="#" class="category__link category__link--wide">
+                    <img
+                      src="./images/categories-image4.png"
+                      alt="Audio"
+                      class="category__photo">
+                  </a>
+                </div>
+                <h3 class="category__title">Audio</h3>
+              </article>
+
+              <article class="category">
+                <div class="category__photos">
+                  <a href="#" class="category__link category__link--wide">
+                    <img
+                      src="./images/categories-image5.png"
+                      alt="Accessories"
+                      class="category__photo"
+                    >
+                  </a>
+                  <a href="#" class="category__link category__link--square">
+                    <img
+                      src="./images/categories-image6.png"
+                      alt="Accessories"
+                      class="category__photo"
+                    >
+                  </a>
+                </div>
+                <h3 class="category__title">Accessories</h3>
+              </article>
+
+              <a href="#" class="categories__btn">View all</a>
+            </div>
+          </section>
+        </div>
+      </div>
+
+      <div class="section-image"></div>
+
+      <div class="container">
+        <div class="main__content">
+          <section class="about-us" id="about-us">
+            <h2 class="section-title about-us__title">About us</h2>
+            <div class="about-us__description">We’re a London-based tech company on a mission to remove barriers between people and technology.</div>
+            <div class="about-us__description">To do this, we’re crafting intuitive, flawlessly connected products that improve our lives without getting in the way.
+              No confusing tech-speak. No silly product names. Just artistry, passion and trust. And products we’re proud to share with our friends and family. Simple.
+            </div>
+            <div class="about-us__description">Because beautiful tech shouldn’t be complicated.</div>
+          </section>
+
+          <section class="contacts" id="contacts">
+            <h2 class="section-title">Contact us</h2>
+            <div class="contacts__wrapper">
+              <form
+                action="/contacts"
+                method="POST"
+                class="contacts__form form"
+              >
+                <input
+                  type="text"
+                  class="form__name "
+                  placeholder="Name"
+                >
+                <input
+                  type="email"
+                  class="form__email"
+                  placeholder="Email"
```

Comment:
Placeholders should have an example of a valid value for the input [Reference](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Placeholders%20should%20give%20examples%20of%20what%20should%20be%20added%2C%20to%20make%20the%20expected%20format%20clear)